### PR TITLE
feat: init the repository

### PR DESCRIPTION
**Pull request by kayman-mk**
# Description

Set up a repository with a dummy Maven project and workflows pushing to Maven Central. For regular updates a script is provided, which automatically creates a PR.

It also provides init branches for the license and codeowners file. If these branches are present, a PR is automatically created. It contains further instructions for the correct setup.

# Verification

Manually verified on the `test` repository.

# Checklist

- [x] My code follows the style guidelines of the project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] add the default workflows, waiting for https://github.com/Hapag-Lloyd/Workflow-Templates/pull/10

### CODE REVIEW

**Comment by kayman-mk** [comment]
default workflows still missing. to be added as soon as https://github.com/Hapag-Lloyd/Workflow-Templates/pull/10 is merged.

File: .github/renovate.json5 [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      matchManagers: ["github-actions"],
+      automerge: false,
+      description: "Do not auto-merge changes in the github-actions as we are referencing theses actions in the `this_*` files which would then create a new PR (circular dependency).",
+    },
+    {
+      matchManagers: ["github-actions"],
+      matchFileNames: [".github/workflows/this_*"],
+      groupName: "GitHub Actions",
+      description: "Group all github-actions packages together as actions use the same digest.",
+    },
```
Comment (PascalFrenz) [inline .github/renovate.json5:16]: I think we do not need these rules here since we do not have the `this_*` files here, correct?

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+    <!-- do not change this version. Done by the workflow before pushing the artifact to Maven Central -->
+    <version>1.0.0-SNAPSHOT</version>
+    <name>zzz</name>
+    <description>aaa</description>
+    <url>https://github.com/Hapag-Lloyd/bbb</url>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
```
Comment (PascalFrenz) [inline pom.xml:18]: Not sure if we want to support Java 8 still by default.. Is that a requirement? If so, why?

File: pom.xml
Comment (PascalFrenz) [inline pom.xml:1]: There are some placeholders here which I found kind of hard to spot, like `bbb`, `xxx`, `yyy` or `zzz`.
I think they should be replaced by something consistent and be easily identifiable as placeholder

**Review by PascalFrenz — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (kayman-mk) [reply pom.xml]: Thought about the https://github.com/Hapag-Lloyd/json-logic-rule-maker project. But 21 should be the default.

File: pom.xml
Comment (kayman-mk) [reply pom.xml:1]: true, should be moved to an `init` branch

**Review by PascalFrenz — APPROVED** [review]

**Pull request merged**